.pre-commit-config.yaml

```diff
@@ -45,7 +45,8 @@ repos:
         exclude: \.svg$
       - id: fix-byte-order-marker
       - id: forbid-submodules
-      #- id: mixed-line-ending
+      - id: mixed-line-ending
+        exclude: ^static/img/.*
```

Comment:
I think this would need to apply to /blog/*/*.png|jpg|gif as well. I am not sure how advanced the exclude clause syntax is, but sure a lot of images are stowed away in those directories as well.

Reply:
> good point, @jbampton , but we do need to add that to the wiki/some readme as well. (nothing to do with this commit)

https://github.com/apache/cloudstack/pull/11680